### ISSUE TITLE
HMW design settings for notifications related to sharing or commenting on challenges?

### ISSUE THREAD

**Issue by fannyBkane** [emfoundation/ce100-app#1493]

**Comment by fannyBkane** [comment]
List of notifications below. Before we rewrite and mockup, @Kimberley-Heath is there anything missing?

**Your interests:**
- Someone has chosen a similar interest
- A new challenge was tagged with one of your interests
- An insight was tagged with one of your interests
- An organisation was tagged with one of your interests

**Your challenges:**
- Someone has chosen an interest matching a tag on your challenge
- A new challenge was tagged with a tag used on your challenge
- An insight was tagged with a tag used on your challenge
- An organisation was tagged with a tag used on your challenge
- Your challenge was shared by someone

**Mentions & comments:**
- Someone has mentioned you in a comment on a challenge
- A new comment has been posted on one of your challenges
- A new comment has been posted on a challenge you have commented on

**Newsletters & Marketing**
- I want to receive cello news
- I want to receive CE100 news

**Comment by Kimberley-Heath** [comment]
@fannyBkane @asimonw can we talk about this a bit more? There are so many notifications listed here that if they are sent individually it may cause a lot of unsubscribes.  If we could cut down the notifications and do more of a personalised 'weekly update' notification of their new matches that week, that might be more effective? Not sure if this can be done but good to talk about it. Thanks!

**Comment by fannyBkane** [comment]
@Kimberley-Heath Yes absolutely, we don't want to overwhelm and cause unsubscribe. I will pop something in the diary to review all together. 

In the meantime, if we don't consider this as a list of notifications, but rather all the opportunities to notify, can you see anything missing?

**Comment by fannyBkane** [comment]
Given the little activity on the app, and time we have, how can we answer the following:

- HMW design a notifications feed? (What's the difference between a notifications feed and the matches page? And HMW design the feed so that it  adds value to the content of the My matches page?)
- HMW enable customisation of the feed?
- HMW design the email notifications (including frequency of occurrence)?
